src/device/simulator.ts

```diff
@@ -127,11 +129,25 @@ export class SimulatorDeviceConnection
       case "state_change": {
         this.state = {
           ...this.state,
-          ...event.data.changes,
+          ...event.data.change,
```

Comment:
This is a bug in the switch to the new API also affecting accelerometer.set_range.